### DISCUSSION TITLE
Technical issues during github Education application

### DISCUSSION THREAD

**Discussion by sofiaPalavecino** [community/community#137208] [GitHub Education]
### Hi everyone,

Today I sent my application for Github education and I encountered myself with two issues while doing so:

1.  It was really uncomfortable to do it. The site, even though it gave me the option, couldn't switch from my frontal camera to my back camera on my phone, so it took me a really long time to get a "decent" picture of a piece of paper.
2. I find it very unhappy that to prove that I'm a student of my university I have to provide a photo of an ID card that I do not possess as the only "Good" option. I and thousands of other students go to public universities that do not provide IDs since, as I said, they are public, they are for all. ¿Is this benefit only for students who pay for higher education and are provided with all the exclusive things, or is this for all the students across the world who want to make their careers grow, even though they don't pay to get silly IDs?
Many universities only give certificates and I'm lucky to even get it on PDF.

Thank you very much.

**Comment by davevad93** [comment]
Hi @sofiaPalavecino ,

I don't have a student ID either but I was able to get my benefits by uploading a screenshot of my last exam results; the problem is that I think there's no upload option anymore.

However there are other options, as stated in the [docs](https://docs.github.com/es/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student), "***Documents that prove your current student status include a picture of your school ID with current enrollment date, class schedule, transcript, and affiliation or enrollment verification letter.***".

So you could try one of the above documents if you don't have an ID.
